CDP4Addin/OfficeRibbon/AddinRibbonPart.cs

```diff
@@ -147,7 +147,7 @@ public override bool GetEnabled(string ribbonControlId, string ribbonControlTag
                 case "CDP4_SelectModelToOpen":
                     return this.session != null;
                 case "CDP4_SelectModelToClose":
-                    return (this.session != null) && (this.session.OpenIterations.Count > 0);
+                    return (this.session?.OpenIterations != null) && (this.session.OpenIterations.Count > 0);
```

Comment:
even though this may be an improvement, i fail to see what it has to do with the rest of the code updates. Please revert, it makes it hard to understand changes made with a specific intent

Reply:
Please revert this as sam asked.

Reply:
Code has been reverted